table/tables/index_test.go

```diff
@@ -54,6 +54,10 @@ func (s *testIndexSuite) TestIndex(c *C) {
 			{
 				ID:   2,
 				Name: model.NewCIStr("test"),
+				Columns: []*model.IndexColumn{
+					{},
+					{},
```

Comment:
because in `GetIndexKey()`, there is a assertion `len(c.idxInfo.Columns) == len(indexedValues)` 
Add empty IndexedColumns so that the following lines won't fail:

```Go
	values := types.MakeDatums(1, 2)
	err = index.Create(txn, values, 1)
```